.gitignore

```diff
@@ -6,10 +6,10 @@ __pycache__/
 
 # C extensions
 *.so
-
 # Distribution / packaging
 .Python
 env/
+.env
```

Comment:
remove this as we are not going to use this method of storing variables.

Reply:
done.